site/docs/numbers/applications.mdx

```diff
@@ -0,0 +1,68 @@
+---
+id: applications
+title: Manage Applications
+slug: /numbers/guides/applications
+description: How to manage applications using the Bandwidth API
+keywords:
+  - bandwidth
+  - applications
+image: ../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const accountId = "{accountId}";
+
+export const Highlight = ({children, color}) => (
+    <span
+      style={{
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+The application is a resource that provides customers with the ability to control messaging (SMS and MMS) and voice using HTTP signaling. This resource identifies the callback information used to notify the customer about voice and messaging events. The application is represented by an application id. The API calls that are used to manage the details of a customer-defined application are accessed through the /accounts/{accountId}/applications resource.
+
+#### For ServiceType of Messaging-V2:
+* MsgCallbackUrl - is deprecated, but required if InboundCallbackUrl is not specified. Updates both InboundCallbackUrl and OutboundCallbackurl
+* InboundCallbackUrl - required for messaging.
+* OutboundCallbackUrl - optional for messaging.
+* InboundCallbackCreds - optional for messaging. Corresponds to InboundCallbackUrl.
+* OutboundCallbackCreds - optional for messaging. Corresponds to OutboundCallbackUrl.
+* RequestedCallbackTypes - optional for messaging. List containing callback types: message-delivered, message-sending, message-failed.
+
+#### For ServiceType of Voice-V2:
```

Comment:
We should probably drop the V2 here since V1 is gone

Reply:
done.